components/org.wso2.carbon.stream.processor.core/src/main/java/org/wso2/carbon/stream/processor/core/event/queue/QueuedEvent.java

```diff
@@ -63,17 +70,19 @@ public long getTimestamp() {
     }
 
     public void setTimestamp(long timestamp) {
-
         this.timestamp = timestamp;
     }
 
     public String getSiddhiAppName() {
-
         return siddhiAppName;
     }
 
     public void setSiddhiAppName(String siddhiAppName) {
-
         this.siddhiAppName = siddhiAppName;
     }
+
+    public String[] getTransportSyncProperties() {
```

Comment:
Same here

Reply:
In here findbugs complaining you need to handle the EI_EXPOSE_REP, EI_EXPOSE_REP2 errors thrown because "May expose internal representation by incorporating reference to mutable object. This code stores a reference to an externally mutable object into the internal representation of the object." Since in the code , no other code tries to change the object , i have suppressed the warning